freqtrade/exchange/exchange.py

```diff
@@ -541,7 +545,10 @@ def refresh_latest_ohlcv(self, pair_list: List[Tuple[str, str]]) -> List[Tuple[s
                     or self._now_is_time_to_refresh(pair, ticker_interval)):
                 input_coroutines.append(self._async_get_candle_history(pair, ticker_interval))
             else:
-                logger.debug("Using cached ohlcv data for %s, %s ...", pair, ticker_interval)
+                logger.debug(
+                    "Using cached ohlcv data for pair %s, interval %s ...",
+                    pair, ticker_interval
```

Comment:
i am personally not a huge fan of wrapping log-lines where it's not neccessary as it unnecessarily increases the file/function length (but it's a real minor nitpick).

Reply:
it's longer than 100 chars if written in one line here

change pls it as you wish if do not understand how exactly it can be written here

Reply:
ah ok, sorry missed that :+1: